Xamarin.Essentials/HapticFeedback/HapticFeedback.android.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Android.Views;
+
+namespace Xamarin.Essentials
+{
+    public static partial class HapticFeedback
+    {
+        internal static bool IsSupported => true;
+
+        static async Task PlatformExecute(HapticFeedbackType type)
+        {
```

Comment:
Should we check for the correct Android permissions here? Or is that not required for haptic feedback?

Reply:
In the manifest file this line should be specified
`<uses-permission android:name="android.permission.VIBRATE"></uses-permission>`